### ISSUE TITLE
Cannot add margins for a dimple plot

### ISSUE THREAD

**Issue by KobaKhit** [ramnathv/rCharts#580]
I am trying to specify the margins for my dimple in R. I am using the way @timelyportfolio did it in a comment here #365 :
@timelyportfolio code
```
d1 <- dPlot(
  Sex~Freq,
  groups = c("Eye"),
  data = as.data.frame(HairEyeColor),
  type = "bubble",
  height = 400,
  width = 600
  ,margins = list(left=80,top=40,right=40,bottom=40)
)
```

However, it does not work. Apparently, the margins parameter is not even used in the layout file located at rCharts/inst/libraries/dimple/layouts/chart.html

To be able to specify margins like in the code above just add to the layout file:
```
{{{
if(opts.margins) {
 mychart.setMargins(opt.margins.left,opt.margins.top,opt.margins.right,opt.margins.bottom)
}
}}}
```
I can submit a pull request.

**Comment by annoporci** [comment]
does margin work? (without an s), that's how it's done with nPlot, e.g.:

n$chart(margin = list(left = 100, right = 100))

**Comment by KobaKhit** [comment]
@annoporci No it doesn't. As I said, there is no code in the layout file that adjusts the margins. It needs to be added, hence, I mentioned the pull request unless @ramnathv or @timelyportfolio  feel like doing it.

**Comment by annoporci** [comment]
@KobaKhit , yes sorry I read your message too quickly and deleted my comment promptly. :-(
